bullet_train-internationalization/lib/bullet_train/internationalization/version.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module BulletTrain
+  module Internationalization
+    VERSION = "0.1.0"
```

Comment:
We may have to update this to 1.3.0

Reply:
Yeah, we should definitely update this to match the version number for everything else.